mesheryctl/cmd/perf.go

```diff
@@ -123,14 +123,14 @@ var perfCmd = &cobra.Command{
 }
 
 func init() {
-	perfCmd.Flags().StringVar(&testURL, "url", "", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testName, "name", StringWithCharset(8), "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&testURL, "url", "", "(required) URL of the endpoint to use for the test")
+	perfCmd.Flags().StringVar(&testName, "name", StringWithCharset(8), "(optional) A memorable name for the test.")
+	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "(optional) Name of the service mesh.")
 	// perfCmd.Flags().StringVar(&testFile, "file", "", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&qps, "qps", "0", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&concurrentRequests, "concurrent-requests", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testDuration, "duration", "30s", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testCookie, "cookie", "meshery-provider=Local (ephemeral session) (free use)", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&loadGenerator, "load-generator", "fortio", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&qps, "qps", "1", "(optional) Queries per second")
```

Comment:
I believe we'll want a default of "0", which cause the behavior of "as many as possible".